Library/Homebrew/test/helper/fixture.rb

```diff
@@ -0,0 +1,13 @@
+module Fixture
+  module_function
+
+  FIXTURE_DIR = File.expand_path(File.join(__FILE__, "..", "..", "fixtures")).freeze
+
+  def path(name)
+    File.join(FIXTURE_DIR, name)
+  end
+
+  def url(name)
+    "file://".concat(path(name))
```

Comment:
String interpolation makes more sense to me here. Additionally, this should live in Cask's test_helper or similar as `fixture_url` and cask-specific fixtures should live in `test/fixtures/cask/`